### ISSUE TITLE
Controlalble horses can't jump

### ISSUE THREAD

**Issue by SickUsernameBrah** [CitizensDev/Citizens2#2105]
The output of command `/version` on my server is: **This server is running Paper version git-Paper-114 (MC:1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)**
The output of command `/version citizens` on my server is: **Citizens version 2.0.26-SNAPSHOT (build 1866)**

If you create a horse or donkey npc you can't jump on the horse.
The horse Attempts to jump but its gets cancelled mid jump. Sometimes while standing still you can jump but that rarely happend while trying this.
The steps I took to recreate this is simply `/npc create horse --type horse` `/npc control` and mount the npc

**Comment by fullwall** [comment]
Yeah this is known - I've been trying to figure out why but having some issues. Remember to always have a saddle on the horse and be max domestication before trying to jump.
